src/server/scripts/Northrend/Ulduar/Ulduar/ulduar.cpp

```diff
@@ -502,11 +410,98 @@ class go_call_tram : public GameObjectScript
     }
 };
 
+class spell_ulduar_teleporter : public SpellScriptLoader
+{
+public:
+    spell_ulduar_teleporter() : SpellScriptLoader("spell_ulduar_teleporter") { }
+
+    enum Teleport_Spells
+    {
+        BASE_CAMP = 64014,
+        GROUNDS = 64032,
+        FORGE = 64028,
+        SCRAPYARD = 64031,
+        ANTECHAMBER = 64030,
+        WALKWAY = 64029,
+        CONSERVATORY = 64024,
+        MADNESS = 64025,
+        SPARK = 65042
+    };
```

Comment:
I'm okay with this because it at least avoids magic numbers, but maybe better in a header file somewhere if one exists?